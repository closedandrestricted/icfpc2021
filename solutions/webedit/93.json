{"vertices": [[12, 49], [12, 47], [9, 41], [14, 45], [14, 41], [17, 53], [17, 59], [16, 76], [20, 47], [22, 46], [22, 49], [14, 64], [25, 49], [18, 45], [26, 122], [50, 18], [7, 38], [48, 15], [28, 51], [46, 10], [22, 130], [11, 144], [40, 14], [44, 15], [38, 98], [16, 145], [32, 78], [27, 51], [33, 55], [21, 61], [49, 12], [42, 12], [47, 72], [45, 14], [55, 41], [45, 12], [29, 71], [29, 54], [46, 12], [39, 111], [17, 157], [33, 63], [17, 157], [20, 160], [31, 57], [57, 78], [44, 121], [19, 150], [52, 74], [54, 56], [58, 44], [23, 149], [61, 74], [55, 128], [25, 145], [57, 70], [65, 80], [26, 153], [26, 165], [30, 153], [54, 56], [80, 77], [61, 121], [66, 77], [74, 83], [78, 67], [83, 145], [76, 114], [84, 67], [78, 84], [84, 151], [84, 108], [84, 97], [76, 124], [85, 66], [68, 171], [83, 167], [97, 71], [91, 61], [86, 169], [87, 61], [86, 126], [101, 123], [96, 60], [99, 127], [109, 64], [101, 58], [99, 120], [100, 120], [108, 61], [96, 56], [96, 118], [99, 53], [90, 103], [104, 51], [103, 123], [99, 124], [99, 121], [104, 128], [108, 135], [101, 52], [104, 56], [113, 76], [113, 73], [106, 54], [96, 133], [105, 145], [107, 49], [96, 111], [115, 143], [119, 77], [117, 79], [128, 84]]}