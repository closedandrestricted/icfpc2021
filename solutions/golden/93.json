{"vertices":[[70,94],[73,98],[81,111],[84,108],[81,85],[83,93],[84,97],[56,121],[81,92],[83,92],[78,86],[44,121],[76,88],[67,118],[39,73],[16,42],[62,121],[13,44],[54,124],[0,31],[33,79],[14,77],[2,35],[11,43],[58,88],[19,73],[46,89],[50,114],[53,113],[37,97],[12,59],[8,37],[50,84],[7,42],[11,64],[6,35],[38,92],[44,106],[11,42],[58,73],[13,64],[44,102],[23,52],[14,60],[34,92],[52,92],[55,60],[19,49],[32,92],[49,78],[40,69],[17,42],[38,98],[54,47],[13,40],[45,86],[58,80],[22,44],[9,47],[20,47],[49,71],[48,97],[55,57],[44,98],[67,91],[22,130],[72,106],[60,73],[26,122],[53,95],[76,102],[51,72],[52,83],[65,82],[43,78],[85,137],[85,120],[49,74],[32,109],[81,120],[35,115],[57,88],[31,45],[40,110],[20,47],[60,72],[49,102],[22,50],[27,44],[63,77],[44,105],[33,68],[54,104],[42,71],[76,114],[20,56],[12,55],[21,59],[21,55],[21,60],[61,110],[58,110],[78,73],[82,73],[73,106],[20,63],[31,52],[74,100],[21,71],[37,53],[93,72],[91,69],[104,68]]}